bot.py

```diff
@@ -558,6 +558,11 @@ async def sendEmbed(self, ctx, townInfo):
 class Gameplay(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.currentlyActiveGames = {}
+        #self.cleanupInactiveGames.start() This should only be run if we have active games in the DB
+
+    def cog_unload(self):
```

Comment:
I don't know if this will run in normal operation - when heroku redeploys I doubt it stops cleanly

Reply:
It was just part of the recommended way to set up the timer so that's what I did